src/interfaces/badge.d.ts

```diff
@@ -0,0 +1,9 @@
+// <reference types="react" />
+export type BadgeType = "primary" | "light";
+
+export interface IBadgeProps {
```

Comment:
it probably should `extends React.HTMLAttributes<HTMLSpanElement>` @kamilmateusiak wdyt?

Reply:
yup, if we use `...restProps `